flask_resty/sorting.py

```diff
@@ -169,3 +170,43 @@ def get_request_field_orderings(self, view):
                 )
 
         return field_orderings
+
+
+class ModelSorting(Sorting):
+    """A sorting component that allows users to specify preconfigured sorting
+    expressions based on the view model.
+
+    For example, to allow users to sort by content length and/or author
+
+        sorting = ModelSorting(
+            author=Post.author
+            content_length=sql.func.length(Post.content)
+        )
+
+    One or both of `author` or `content_length` can be formatted in the `sort_arg`
+    request parameter to determine the sort order. For example, users can sort requests
+    by `name` ascending and `date` descending by making a ``GET`` request to::
+
+        /api/posts?sort=author,-content_length
+
+    :param str default: If provided, specifies a default sort order when the
+        request does not specify an explicit sort order.
+
+    :param dict kwargs: A mapping from sort names to order_by expressions
+    """
+
+    def __init__(self, default=None, **kwargs):
+        super().__init__(*kwargs.keys(), default=default)
```

Comment:
maybe this should also allow for field name positional args, to pass through to Sorting?